.travis.yml

```diff
@@ -11,3 +11,9 @@ script:
     fi
   - npm run coveralls
   - npm run lint
+  - >
+    if [[ $(node --version | cut -c 2-) < 6 ]]; then
```

Comment:
Could you please change the version test. Either use `grep` or change the previous test to use `cut`. `cut` is probably better.